schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -170,6 +170,11 @@
       <summary>TODO</summary>
       <description>TODO</description>
     </key>
+    <key type="b" name="show-notification-badges">
```

Comment:
It might be nice to have this tri-state:

- `yes`
- `no`
- `when-not-in-dnd-mode` https://github.com/micheleg/dash-to-dock/issues/1329